.github/pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+<!--
+BEFORE YOU CREATE A PULL REQUEST:
+
+Ensure you have read over
+
+CONTRIBUTING - https://github.com/puppetlabs/puppetlabs-iis/blob/master/CONTRIBUTING.md
+
+We provide defined guidance (as such, we strongly adhere to it).
+
+A summary of our expectations:
+ - A ticket was created at https://tickets.puppet.com/secure/CreateIssue!default.jspa, or you have an existing ticket #
```

Comment:
Per the docs at https://confluence.atlassian.com/jirakb/creating-issues-via-direct-html-links-159474.html ... something like https://tickets.puppetlabs.com/secure/CreateIssueDetails!init.jspa?pid=10707&components=15066 should auto-start in MODULES with an IIS component (components available at https://tickets.puppetlabs.com/rest/api/2/project/MODULES/components)

However, I'm seeing an error `Error rendering 'issue-templates-customfield'. Please contact your JIRA administrators.`

Reply:
Maybe @austinboyd can point us in the right direction?

Reply:
You're missing a requirement for issuetype in your URI. This example has Bug as the issuetype, let me know if you need help identifying other issuetypes. https://tickets.puppetlabs.com/secure/CreateIssueDetails!init.jspa?pid=10707&issuetype=1&components=15066

Reply:
I copied what's in the CONTRIBUTING.md doc, since this point of this is to highlight the most important things people miss. If the url is wrong, it needs to be updated there too.

Reply:
Thanks @austinboyd - missed the `issuetype` requirement.  That looks right to me.

Reply:
@jpogran original url isn't wrong per se - it's more that we can at least start users in the appropriate project with the right component.

That said, I'd be in favor of updating in the README as well.